index.html

```diff
@@ -0,0 +1,457 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+  
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+
+
+      <div id="mySidenav" class="sidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+        <a class="nav-link" href="#projects">Portoflio</a>
+        <a class="nav-link" href="#about-me">About</a>
+        <a href="#contact">Contact</a>
+    
+      </div>
```

Comment:
- [ ] Please make sure to use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) in your tags, and since this element is a menu. So, kindly make it a `<nav>` tag instead of `<div>`, and each link is supposed to be inside a `<ul>` and `<li>` tags.
- [ ] Also, kindly make sure to make it possible when the user clicks (taps) the hamburger button it disappears, and the close 'X' button appears instead, not the both at the same time. Please work on that.
- [ ] Please make sure when a user clicks (taps) a link within the list of the mobile menu, the menu should disappear. Currently, it remains open. So, please work on this issue as well.
**_Please see the above comment about the design so it makes sense with these requirements_**